docs/cep/table/table-indexes.md

```diff
@@ -3,34 +3,172 @@ sidebar_position: 30
+
+| Property    | Description                                                | Default Value |
+| ----------- | ---------------------------------------------------------- | ------------- |
+| unique      | If true, then create a unique index.                       | false         |
+| sparse      | If true, then create a sparse index.                       | false         |
+| deduplicate | If false, the deduplication of array values is turned off. | false         |
+
+Example:
+
+```sql
+-- Creates a persistent index named `SamplePersistentIndex` on `SampleGDNTable` with following properties {unique=true, sparse=true, deduplicate=true}.
+CREATE UNIQUE INDEX SamplePersistentIndex ON TABLE SampleGDNTable WITH(type="persistent", sparse="true", deduplicate="true") (sensorId);
+```
+
+### Skiplist Index Properties
+
+The following properties apply to [skiplist indexes](../../collections/indexing/index#skiplist-index). They are all optional.
+
+| Property    | Description                                                | Default Value |
+| ----------- | ---------------------------------------------------------- | ------------- |
+| unique      | If true, then create a unique index.                       | false         |
+| sparse      | If true, then create a sparse index.                       | false         |
+| deduplicate | If false, the deduplication of array values is turned off. | false         |
+
+Example:
+
+```sql
+-- Creates a skiplist index named `SampleSkiplistIndex` on `SampleGDNTable` with properties {unique=true, sparse=true, deduplicate=true}.
+CREATE UNIQUE INDEX SampleSkiplistIndex ON TABLE SampleGDNTable WITH(type="skiplist", sparse="true", deduplicate="true") (sensorId);
+```
 
+### TTL Index Properties
+
+The following properties apply to [ttl indexes](../../collections/indexing/ttl-indexes). They are all optional.
+
+| Property    | Description                                                | Default Value |
+| ----------- | ---------------------------------------------------------- | ------------- |
+| expireAfter      | The number of seconds time after which the documents count as expired. The countdown starts at document creation.                     | 3600         |
+
+Example:
+
+```sql
 -- Creates a ttl index named `SampleTTLIndex` on `SampleGDNTable` with property {expireAfter=3600}.
 CREATE INDEX SampleTTLIndex ON TABLE SampleGDNTable WITH(type="ttl", expireAfter="3600") (sensorId);
 ```
+
+## Example 1
+
+```sql
+CREATE TABLE SampleGDNTable (sensorId string, temperature double);
+CREATE UNIQUE INDEX SamplePersistentIndex ON TABLE SampleGDNTable WITH(type="persistent", sparse="true") (sensorId);
+```
+
+This example creates a table named `SampleGDNTable` with two fields: `sensorId` of type `string` and `temperature` of type `double`. The second statement creates a unique, persistent index named `SamplePersistentIndex` on the `SampleGDNTable` table with `sensorId` as the field to index.
+
+## Example 2
```

Comment:
@StoyanTc - Is the explanation for this example correct? Also, what sort of index is created here?

Reply:
@diana-macrometa - Example 2 is invalid.